docs/sources/release-notes/v2-5.md

```diff
@@ -5,50 +5,75 @@ weight: 77
+```
 
-Here is a list of limit defaults that have changed in v2.5.0:
+In 2.5.0 it can **only be defined in the `limits_config` section**. 
 
-| config | new default | old default |
-| --- | --- | --- |
-| param name here | new value | old value |
+**Loki will fail to start if you do not remove the `split_queries_by_interval` config from the `query_range` section.**
 
-## Bug fixes
+Additionally, it has a new default value of `30m` rather than `0`.
 
-### 2.5.0 bug fixes
-2.0.0 fixes these bugs:
+The CLI flag is not changed and remains `querier.split-queries-by-interval`.
 
-- [PR XXXX](https://github.com/grafana/loki/pull/XXXX) **github-handle**: Description of the bug fix here.
+### More parallelism by default
 
-## Security fixes
+We continue to move defaults in Loki to leverage parallelism in all configurations, including the single binary.
 
-**Do we need this section?**
+In 2.5.0 all queries will be split and sharded by default, this will likely result in increased memory and CPU usage for Loki processes during queries if you didn’t previously have these values enabled.
 
-### 2.5.0 security fixes
+## Bug fixes
 
-Version 2.5.0 contains X security-related fix:
+### 2.5.0 bug fixes
 
-* [PR XXXX](https://github.com/grafana/loki/pull/XXXX) Description of security-related fix here.
+2.5.0 fixes numerous bugs, The [CHANGELOG](https://github.com/grafana/loki/blob/main/CHANGELOG.md) has a more detailed list.
+
+A summary of some of the more important fixes:
+
+* Query correctness:
+  * [5474](https://github.com/grafana/loki/pull/5474) **cyriltovena**: Disable sharding of count/avg when labels are mutated
+  * [5444](https://github.com/grafana/loki/pull/5444) **cyriltovena**: Do not insert missing point when sharding
+  * [5423](https://github.com/grafana/loki/pull/5423) **cyriltovena**: Correctly sets hash value for headblock iterator
+  * [5289](https://github.com/grafana/loki/pull/5289) **cyriltovena**: Fixes log deduplication when mutating Labels using LogQL
+  * [5006](https://github.com/grafana/loki/pull/5006) **sandeepsukhani**: fix splitting of queries when step is larger than split interval
+* [A deadlock in the Azure blob client.](https://github.com/grafana/loki/pull/5413)
+* [A fix in live tailing](https://github.com/grafana/loki/pull/5334) that could lead to exploding memory usage.
+* [A fix for the ruler when trying to use Basic Auth for remote write.](https://github.com/grafana/loki/pull/5144)
+* Query cancellation:
+  * [5113](https://github.com/grafana/loki/pull/5113) **kavirajk**: Fix cancel issue between Query Frontend and Query Schdeduler
+  * [5080](https://github.com/grafana/loki/pull/5080) **kavirajk**: Handle `context` cancellation in some of the `querier` downstream requests
+  * [5075](https://github.com/grafana/loki/pull/5075) **cyriltovena**: Fixes a possible cancellation issue in the frontend
+* [An issue where retention didn't cleanup the index completely.](https://github.com/grafana/loki/pull/4741)
```

Comment:
Changes in this suggestion are meant to use the same formatting as the v2.4 release notes.

```suggestion
* Query correctness:
  * [PR 5474](https://github.com/grafana/loki/pull/5474) **cyriltovena**: Disable sharding of count/avg when labels are mutated
  * [PR 5444](https://github.com/grafana/loki/pull/5444) **cyriltovena**: Do not insert missing point when sharding
  * [PR 5423](https://github.com/grafana/loki/pull/5423) **cyriltovena**: Correctly sets hash value for headblock iterator
  * [PR 5289](https://github.com/grafana/loki/pull/5289) **cyriltovena**: Fixes log deduplication when mutating Labels using LogQL
  * [PR 5006](https://github.com/grafana/loki/pull/5006) **sandeepsukhani**: Fix splitting of queries when step is larger than split interval
* [PR 5413](https://github.com/grafana/loki/pull/5413) **owen-d**: A deadlock in the Azure blob client
* [PR 5334](https://github.com/grafana/loki/pull/5334) **kavirajk**: A fix in live tailing that could lead to exploding memory usage
* [PR 5144](https://github.com/grafana/loki/pull/5144) **owen-d**: A fix for the ruler when trying to use Basic Auth for remote write
* Query cancellation:
  * [PR 5113](https://github.com/grafana/loki/pull/5113) **kavirajk**: Fix cancel issue between query-frontend and query scheduler
  * [PR 5080](https://github.com/grafana/loki/pull/5080) **kavirajk**: Handle `context` cancellation in some of the `querier` downstream requests
  * [PR 5075](https://github.com/grafana/loki/pull/5075) **cyriltovena**: Fixes a possible cancellation issue in the frontend
* [PR 4741](https://github.com/grafana/loki/pull/4741) **sandeepsukhani**: An issue where retention didn't cleanup the index completely
```